app/main/helpers/suppliers.py

```diff
@@ -0,0 +1,28 @@
+import os
+import json
+
+
+def load_countries():
+    helpers_path = os.path.abspath(os.path.dirname(__file__))
+    countryfile = os.path.join(helpers_path, '../../static/location-autocomplete-canonical-list.json')
+    with open(countryfile) as f:
+        return (json.load(f))
+
+
+def get_country_name_from_country_code(country_code):
+    if country_code and country_code == 'gb':
+        # We need to support the old country code style ('gb') until after the existing country code data we have in the
+        # database has been updated by a script we'll run.
+        return 'United Kingdom'
```

Comment:
If we plan to remove this after the migration, can we add a TODO here (with 'remove after x') and/or capture it on another ticket?